cpp/cmake_modules/FindBrotli.cmake

```diff
@@ -69,6 +69,7 @@ if(BROTLI_ROOT)
             PATH_SUFFIXES ${INCLUDE_PATH_SUFFIXES}
             NO_DEFAULT_PATH)
 else()
+  find_package(PkgConfig QUIET)
```

Comment:
Is this needed?
We have `include(FindPkgConfig)` in `cpp/CMakeLists.txt` for `pkg_check_modules()`.

Reply:
The find modules for static libraries that are linked to are now also used by `ArrowConfig.cmake`. I could have added this line there directly, but I prefer modules to be self-sufficient.

Reply:
I see.
I'll review this again.